lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -563,7 +566,9 @@ export async function addReviewers(
       throw new Error(REPOSITORY_NOT_FOUND);
     }
 
-    const reviewersSet = new Set([...pr.reviewers, ...reviewers]);
+    // TODO: can `reviewers` be undefined?
```

Comment:
We previously did a bunch of work to annotate all TODO's with issues :)

Reply:
I'll go through the code in another PR and add an issue number to track.